src/components/PlioQuestion.vue

```diff
@@ -151,7 +152,9 @@ export default {
         "config": this.progressBarInfo['config'],
         "progressPercent": 0,
         "totalQuestions": this.progressBarInfo['totalQuestions']
-      }
+      },
+      questionText: this.plioQuestion.item.question.text,
+      optionText: this.plioQuestion.item.question.options,
```

Comment:
Saving the `questions` and `options` information as data properties so they can be referenced in the `v-html` directive as explained above